pkg/query/querier_test.go

```diff
@@ -245,139 +244,77 @@ func TestStoreMatches(t *testing.T) {
 	}
 }
 
-func expandSeries(t testing.TB, it storage.SeriesIterator) (res []sample) {
+func expandSeries(t testing.TB, it storage.SeriesIterator) (res []testutil.Sample) {
 	for it.Next() {
 		t, v := it.At()
-		res = append(res, sample{t, v})
+		res = append(res, testutil.Sample{t, v})
```

Comment:
Why move that trivial type into an extra package. It makes everything more verbose basically.

Reply:
yes, but do you have any alternative?

Reply:
What's the issue with keeping it local. We have the same type in various places across test files of TSDB and Prometheus. The type duplication is usually not an issue and using test helpers tend to be sufficiently different to not be worth it.

Really not a fan of making `testutil` even more of a catch-all sink than it already is.

Reply:
Same with the `testutil.StoreSeriesServer` – seems like it is never used outside of the current package, so what's the concern with leaving it there?

One issue with extracting these mocks in general is that other importers may have different requirements in terms of error injection or what not and those mocks slowly get a lot of odd injection mechanisms to fit all use cases. Duplicating and adapting for the exact local needs always seemed a lot of less overall hassle to me.